Automattic/wp-calypso
# Add color scheme ocean

One part of #47036

#### Changes proposed in this Pull Request

* Add ocean color scheme

#### Testing instructions

* Visit /me/account
* Choose ocean and save
* Try with flag nav unification on and off

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso (Flag off):
![2020-11-11_11-35](https://user-images.githubusercontent.com/937354/98844517-095c7e80-2412-11eb-9905-51e897702e63.png)


Calypso (Flag on):
![2020-11-11_11-36](https://user-images.githubusercontent.com/937354/98844576-1e391200-2412-11eb-8905-031089b2c9b0.png)


Wp-Admin:
![2020-11-11_11-34](https://user-images.githubusercontent.com/937354/98844419-e7fb9280-2411-11eb-9b68-ea1657dd7aa8.png)


### Reference Issues

* Main Issue #45435
  * Color Schemes Issue + Direction  #45675
    * Nav unification: Port over WP Admin color schemes as new options in Calypso /me section. #47036
    * Nav Unification: Rename the Ocean color scheme to Aquatic; update user preferences accordingly #47256
